airbyte-integrations/connectors/source-typeform/source_typeform/spec.json

```diff
@@ -0,0 +1,23 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/typeform",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Shopify Source CDK Specifications",
+    "type": "object",
+    "required": ["token", "start_date"],
+    "additionalProperties": true,
+    "properties": {
```

Comment:
also alphabetise.

Reply:
ah no. I was talking about `start_date` being before `token`.

Same as the above.